package.json

```diff
@@ -9,34 +9,32 @@
     "homepage": "https://github.com/bazelbuild/rules_nodejs",
     "license": "Apache-2.0",
     "devDependencies": {
-        "@bazel/bazel": "^0.21.0",
+        "//": "need release candidate of Bazel to pick up https://github.com/bazelbuild/bazel/pull/6980",
+        "@bazel/bazel": "^0.23.0-rc1b",
         "@bazel/buildifier": "^0.20.0",
-        "@bazel/typescript": "^0.22.1",
+        "@bazel/typescript": "0.23.2",
```

Comment:
0.23.3

Reply:
The difference is just the ngtsc hook in ts_library, no need to update to latest